### PR TITLE
Created webpages for metaomics SIG

### PR DESCRIPTION
**Pull request by CarsonJM**
@netlify /special-interest-groups/meta-omics

### CODE REVIEW

**Review by erikrikarddaniel — COMMENTED** [review]
Just two comments. I suppose it's a little early to set all the details and that we'll need to discuss meeting schedule and focus before deciding.

File: src/content/special-interest-groups/meta-omics/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+# Introduction
+
+Meta-omics methods aim to simultaneously analyse the entire multi-species potential and realised functional capacity of a biological sample. In other words to taxonomically and functionally characterise all organisms and their activity in a community.
+
+The initial aims of this interest group will be:
+1) helping coordinate compatibility between existing and potential meta-omic nf-core pipelines
+2) implementing semi-'chaining' of sequential pipelines
+3) developing standard parameters for different sub-disciplines (i.e. microbiome analysis) for each pipeline
+4) documenting usage of 'first level' meta-omic pipelines followed by complementary analyses with "secondary" pipelines such as #funcscan, #phageannotator, #phyloplace, #rnaseq, #differentialabundance etc.
```
Comment (jfy133) [inline src/content/special-interest-groups/meta-omics/index.md:33]:
```suggestion

1. helping coordinate compatibility between existing and potential meta-omic nf-core pipelines
2. implementing semi-'chaining' of sequential pipelines
3. developing standard parameters for different sub-disciplines (i.e. microbiome analysis) for each pipeline
4. documenting usage of 'first level' meta-omic pipelines followed by complementary analyses with "secondary" pipelines such as #funcscan, #phageannotator, #phyloplace, #rnaseq, #differentialabundance etc.
```

File: src/content/special-interest-groups/meta-omics/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Depending on interst/activity, other potential activities could include:
+1) benchmarking meta-omic pipelines using relevant datasets (i.e. CAMI2), or reads generated using #readsimulator
+2) adding nf-tests to meta-omic pipelines
+3) reducing functional redundancy between pipelines
```
Comment (jfy133) [inline src/content/special-interest-groups/meta-omics/index.md:38]:
```suggestion
1. benchmarking meta-omic pipelines using relevant datasets (i.e. CAMI2), or reads generated using #readsimulator
2. adding nf-tests to meta-omic pipelines
3. reducing functional redundancy between pipelines
```

File: src/content/special-interest-groups/meta-omics/meetings.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'Meetings'
+subtitle: 'When and where the Meta-Omics group meets'
+groupName: Meta-omics
```
[the remark this answers is not in the document]
Comment (jfy133) [reply src/content/special-interest-groups/meta-omics/meetings.md:4]: I would go lower case, I've never seen `Omics`

File: src/content/special-interest-groups/meta-omics/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+title: Introduction
+subtitle: 'Investigating biological community composition and dynamics using metagenomic, metatranscriptomic, and metaproteomic approaches.'
```
Comment (jfy133) [inline src/content/special-interest-groups/meta-omics/index.md:3]:
```suggestion
subtitle: 'Community of researchers investigating biological community composition and dynamics using metagenomic, metatranscriptomic, and metaproteomic approaches.'
```

File: src/content/special-interest-groups/meta-omics/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+1) benchmarking meta-omic pipelines using relevant datasets (i.e. CAMI2), or reads generated using #readsimulator
+2) adding nf-tests to meta-omic pipelines
+3) reducing functional redundancy between pipelines
+
+# Activities
+
+When were we thinking of having meetings?
+Are these going to be development meetings, or planning meetings?
+What platform should we post meetings on?
+Should we create a project board that we can post tasks on?
```
Comment (jfy133) [inline src/content/special-interest-groups/meta-omics/index.md:45]:
```suggestion
Initial activities will involve a monthly meeting with developers and users, to identify the most relevant problems related meta-omics with nf-core pipelines. Future activities may include presentations and discussions from topic-experts, as well as  tutorials, workshops, and demo-days of nf-core pipelines.
```

What about something like that?

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
@CarsonJM was this meant to be closed?

**Pull request closed without merging**